extra/mondoo-googleworkplace-incident-response.mql.yaml

```
@@ -63,23 +63,23 @@ packs:
       - asset.platform == "googleworkspace" || asset.platform == "google-workspace"
     queries:
       - uid: mondoo-googleworkspace-incident-response-domain
-        title: Retrieve Google Workspace Domains
+        title: Google Workspace Domains
         mql: googleworkspace.domains { domainName isPrimary verified }
       - uid: mondoo-googleworkspace-incident-response-user-mfa-status
-        title: Retrieve Google Workspace Users MFA status
+        title: Google Workspace Users MFA status
```

```suggestion
        title: Google Workspace users' MFA status
```

If it's multiple users